### PR TITLE
DS-2304: NextJS PoC - ontario-table page

### PR DESCRIPTION
**Pull request by MeaghanCarrieres**
This PR adds a test page for the `<ontario-table>` component and fixes an asset path issue affecting the highlight indicator icon.

## What's included

- Adds `<ontario-table>` page to NextJS PoC to test for the following prop variants:
    - `caption`
    - `table-columns`
    - `table-data`
    - `zebra-stripes`
    - `condensed`
    - `full-width`
    - Negative / edge case examples (invalid or missing data)
- Asset path fix
    - Updates the highlight indicator SVG in `<ontario-table>` to use the shared `getImageAssetSrcPath()` helper instead of directly calling `getAssetPath()`.
    - This ensures the asset resolves correctly across environments (Stencil, Angular, React/Next) and avoids runtime errors when getAssetPath() does not have a valid asset base path.
- Scrollbar logic improvements
    - Small improvements were made to the scrollbar handling logic to make the component more resilient:
        - Added guards around DOM references used by applyScrollbar
        - Prevents runtime errors if elements are not available during initialization
        - Improves stability during hydration and resize events

### CODE REVIEW

File: packages/app-nextjs/src/app/components/ontario-table/page.tsx [hunk 1]
```diff
@@ -0,0 +1,654 @@
...
+							{ data: { region: 'Central', count: '1,248' } },
+							{ data: { region: 'East', count: '982' } },
+							{ data: { region: 'North', count: '412' } },
+							{ data: { region: 'South West', count: '1,031' } },
+							{ data: { region: 'Toronto', count: '2,543' } },
+							{ data: { region: 'Ottawa', count: '1,876' } },
+						]}
+					></OntarioTable>
+
+					<h3>`zebraStripes`` enabled</h3>
```
Comment (erincarrieresontario) [inline packages/app-nextjs/src/app/components/ontario-table/page.tsx]:
```suggestion
					<h3>`zebraStripes` enabled</h3>
```

**Review by erincarrieresontario — APPROVED** [review]
Looks great @MeaghanCarrieres , just one small thing !
